gauss_imputation_known_params_demo.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        plt.ylabel("Dimension of Data")
</pre><p>Data -&gt; data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done